### PR TITLE
Add provider equinix and module_name in Equinix Metal TF configs

### PR DESCRIPTION
**Pull request by codinja1188**

### CODE REVIEW

File: examples/fabric-port-connection/versions.tf [hunk 1]
```diff
@@ -4,11 +4,14 @@ terraform {
...
+      version = "~> 1.14"
     }
     aws = {
       source  = "hashicorp/aws"
       version = ">= 3.64"
     }
   }
+  provider_meta "equinix" {
+    module_name = "equinix-fabric-connection-aws/fabric-port-connection"
+  }
```
Comment (ocobles) [inline examples/fabric-port-connection/versions.tf]: Examples don't need to define provider_meta, it's just for the module itself
```suggestion
```

File: examples/service-token-metal-to-aws-connection/versions.tf [hunk 2]
```diff
@@ -4,11 +4,14 @@ terraform {
...
+      version = "~> 1.14"
     }
     aws = {
       source  = "hashicorp/aws"
       version = ">= 3.64"
     }
   }
+  provider_meta "equinix" {
+    module_name = "equinix-fabric-connection-aws/service-token-metal-to-alibaba-connection"
+  }
```
Comment (ocobles) [inline examples/service-token-metal-to-aws-connection/versions.tf]:
```suggestion
```

File: examples/network-edge-device-connection/versions.tf [hunk 3]
```diff
@@ -4,11 +4,14 @@ terraform {
...
+      version = "~> 1.14"
     }
     aws = {
       source  = "hashicorp/aws"
       version = ">= 3.64"
     }
   }
+  provider_meta "equinix" {
+    module_name = "equinix-fabric-connection-aws/network-edge-device-connection"
+  }
```
Comment (ocobles) [inline examples/network-edge-device-connection/versions.tf]:
```suggestion
```

**Review by ocobles — CHANGES_REQUESTED** [review]
Please, review requested changes

File: examples/fabric-port-connection/versions.tf (hunk 1 above)
Comment (codinja1188) [reply examples/fabric-port-connection/versions.tf]: updated

File: examples/network-edge-device-connection/versions.tf (hunk 3 above)
Comment (codinja1188) [reply examples/network-edge-device-connection/versions.tf]: updated

File: examples/service-token-metal-to-aws-connection/versions.tf (hunk 2 above)
Comment (codinja1188) [reply examples/service-token-metal-to-aws-connection/versions.tf]: updated

**Review by ocobles — APPROVED** [review]
👍🏼

**Pull request merged**